app/controllers/DataSetController.scala

```diff
@@ -520,4 +530,32 @@ Expects:
       case _             => Messages("dataSet.notFoundConsiderLogin", dataSetName)
     }
 
+  @ApiOperation(hidden = true, value = "")
+  def segmentAnythingEmbedding(organizationName: String,
+                               dataSetName: String,
+                               dataLayerName: String): Action[SegmentAnythingEmbeddingParameters] =
+    sil.SecuredAction.async(validateJson[SegmentAnythingEmbeddingParameters]) { implicit request =>
+      for {
+        _ <- bool2Fox(conf.Features.segmentAnythingEnabled) ?~> "segmentAnything.notEnabled"
+        _ <- bool2Fox(conf.SegmentAnything.uri.nonEmpty) ?~> "segmentAnything.noUri"
+        // - <- bool2Fox(request.body.boundingBox.depth == 1) ?~> "segmentAnything.bboxNotFlat"
+        dataset <- dataSetDAO.findOneByNameAndOrganizationName(dataSetName, organizationName) ?~> notFoundMessage(
+          dataSetName) ~> NOT_FOUND
+        dataSource <- dataSetService.dataSourceFor(dataset) ?~> "dataSource.notFound" ~> NOT_FOUND
+        usableDataSource <- dataSource.toUsable ?~> "dataSet.notImported"
+        dataLayer <- usableDataSource.dataLayers.find(_.name == dataLayerName) ?~> "dataSet.noLayers"
+        datastoreClient <- dataSetService.clientFor(dataset)(GlobalAccessContext)
+        targetMaxBbox: BoundingBox = request.body.boundingBox / request.body.mag
+        // _ <- bool2Fox(targetMaxBbox.dimensions == Vec3Int(1024, 1024, 1)) ?~> s"Target-mag bbox must be sized 1024×1024×1, got ${targetMaxBbox.dimensions}"
```

Comment:
@fm3 I commented the two assertions here to support other viewports. Could you clean this up so that the assertions make sense again? :) E.g., sort the dimensions and compare against `[1, 1024, 1024]`?